repo2docker/buildpacks/base.py

```diff
@@ -246,7 +246,7 @@ def get_labels(self):
         """
         Docker labels to set on the built image.
         """
-        return {}
+        return getattr(self, 'labels', {})
```

Comment:
`getattr` makes my Python senses tingle :-/

Can we change this to `return self.labels` and set `self.labels = {}` in `__init__` instead?